extension/natives/takedamageinfo.cpp

```diff
@@ -503,6 +566,10 @@ void setup(std::vector<sp_nativeinfo_t>& natives) {
 	sp_nativeinfo_t list[] = {
 		{"CTakeDamageInfo.CTakeDamageInfo", CTakeDamageInfo_Ctor},
 
+		{"GetGlobalDamageInfo", GetGlobalDamageInfo},
+
+		{"CTakeDamageInfo.Set", Set},
```

Comment:
I think it should be called `CTakeDamageInfo.Init`, because it has no default parameters. While I would expect at least one mandatory parameter for a function named `Set`